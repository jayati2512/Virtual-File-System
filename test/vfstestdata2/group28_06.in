createvfs myvfs 2048
createvfs myvfs/ 1024
createvfs myvfsssssssssssssssssssssssssssss
create myvfs 1024
mountvfs
mountvfs myvfs
makedir /dir1
makedir / dir1
makedir /dir1 dir2
deletedir
deletedir /home/dir3
addfile /dir1 out.txt /usr/include/a.txt
addfile /dir1/dir2 test.jpg /usr/include/b.txt
makedir / home
movedir /home 
movedir /dir1 /dir1/dir2
movedir /dir1 out.txt
listfile /dir1/dir2/test.jpg /tmp/temp.txt 
unmount myvfs 
