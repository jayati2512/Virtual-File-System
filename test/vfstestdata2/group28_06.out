createvfs_FAILURE INVALID_SIZE
createvfs_FAILURE INVALID_CHARACTER_IN_NAME
createvfs_FAILURE VFS_LABEL_TOO_LARGE
createvfs_SUCCESS
mountvfs_FAILURE VFS_INSUFFICIENT_ARGUMENTS
mountvfs_SUCCESS
makedir_FAILURE VFS_INSUFFICIENT_ARGUMENTS
makedir_SUCCESS
makedir_SUCCESS
deletedir_FAILURE VFS_INSUFFICIENT_ARGUMENTS
deletedir_FAILURE CANNOT_FIND_SPECIFIED_PATH_OR_DIR
addfile_SUCCESS
addfile_SUCCESS
makedir_SUCCESS
movedir_FAILURE VFS_INSUFFICIENT_ARGUMENTS
movedir_FAILURE CANNOT_MOVE_PARENT_TO_CHILD_DIR
movedir_FAILURE DESTINATION_CANNOT_BE_FILE
list_FAILURE NOT_A_TEXT_FILE
unmount_SUCCESS 
